Samples/HelloWorld/SiloTests/OrleansConfigurationForTesting.xml

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<OrleansConfiguration xmlns="urn:orleans">
-  <Globals>
-    <StorageProviders>
-      <Provider Type="Orleans.Storage.MemoryStorage" Name="MemoryStore" />
-      <Provider Type="Orleans.Storage.MemoryStorage" Name="Default" />
-      <!--<Provider Type="Orleans.Storage.AzureTableStorage" Name="AzureStore"/>-->
-    </StorageProviders>
-    <SeedNode Address="localhost" Port="22222"/>
-    <Messaging ResponseTimeout="30s"/>
-  </Globals>
-  <Defaults>
-    <Networking Address="localhost" Port="22222"/>
-    <ProxyingGateway Address="localhost" Port="40000" />
-    <Tracing DefaultTraceLevel="Info" TraceToConsole="false" TraceToFile="{0}-{1}.log" PropagateActivityId="false" BulkMessageLimit="1000">
-       <TraceLevelOverride LogPrefix="Application" TraceLevel="Info" />
-       <!--
-       <TraceLevelOverride LogPrefix="Runtime.Dispatcher" TraceLevel="Verbose" />
-       <TraceLevelOverride LogPrefix="AssemblyLoader.Silo" TraceLevel="Warning" />
-       -->
-    </Tracing>
-    <Statistics MetricsTableWriteInterval="30s" PerfCounterWriteInterval="30s" LogWriteInterval="300s" WriteLogStatisticsToTable="true" StatisticsCollectionLevel="Info"/>
-  </Defaults>
-</OrleansConfiguration>
-
```

Comment:
I see you removed xml config, did you add programmatic config , or we are using default config now?